docs/poetry.lock

```diff
@@ -0,0 +1,1374 @@
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+
```

Comment:
Is this part of the commit or accidental?

Reply:
It's part of the commit (update to the latest version). We now include `poetry.lock` in all projects ([context](https://github.com/scylladb/sphinx-scylladb-theme/issues/849)).

Unless contributors manually run `make update`, the `poetry.lock` file will remain static and won’t be changed in other pull requests.

Reply:
In the future, consider doing this as a separate PR or at least as a separate commit, so it doesn't get lost in the noise.